kitsune/users/jinja2/users/auth.html

```diff
@@ -50,7 +50,7 @@ <h1 class="sumo-page-subheading sumo-header-medium">{{ _('Continue with your Moz
                   <div class="trouble-text">
                     <p class="help-text">
                       {{ _('Having trouble signing in? ')}}
-                      <a href="{{ url('questions.aaq_step3', product_key='mozilla-account') }}">{{ _('Contact support') }}</a>
+                      <a href="{{ url('questions.aaq_step3', product_key='mozilla-account') }}?skip_auth=True">{{ _('Contact support') }}</a>
```

Comment:
Nit. The case for the value given to `skip_auth` doesn't matter, in fact any non-empty string works, so I think it's simpler to just use lowercase.
```suggestion
                      <a href="{{ url('questions.aaq_step3', product_key='mozilla-account') }}?skip_auth=true">{{ _('Contact support') }}</a>
```

Reply:
Sure thing.